cmd/openshift-install/main.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/openshift/installer/pkg/asset"
 	"github.com/openshift/installer/pkg/asset/stock"
 	"github.com/openshift/installer/pkg/destroy"
+	_ "github.com/openshift/installer/pkg/destroy/libvirt"
```

Comment:
why add this to main.go and not pkg/destroy/destroyer.go ?

Reply:
> why add this to main.go and not pkg/destroy/destroyer.go ?

The libvirt package needs access to `destroy.Registry`, and we don't want an import cycle.